GlideRecord/List of Child Incidents/listofchildIncident.js

```diff
@@ -0,0 +1,11 @@
+var parentID = '9c573169c611228700193229fff72400'; // This should be stored in sys_property instead of hardcoding the sys_id. 
+var inc = new GlideRecord('incident');
+inc.addEncodedQuery('parent_incident='+parentID); // sys_id of parent Incident
+inc.query();
+
+gs.info('List of Child Incidents of:'+inc.getDisplayValue('parent_incident'));
```

Comment:
You should have the if condition to validate the query, as I posted in my previous comment:
```
if(inc.next()) {
  gs.info('List of child incidents of ' + inc.getDisplayValue('parent_incident'));
}
```